### PR TITLE
Replace fixed version in README

### PR DESCRIPTION
**Pull request by JumpingYang001**
@jeffaco , I think you are waiting this PR to not to update release version in README frequently. :)
Could you help to review this PR? thanks very much!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,16 +33,16 @@ downloads illustrates the package types for OMI:
...
-Linux    | RPM     | x64          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x64.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x64.rpm)
-Linux    | Debian  | x86          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x86.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x86.deb)
-Linux    | Debian  | x86          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x86.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x86.deb)
-Linux    | RPM     | x86          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x86.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x86.rpm)
-Linux    | RPM     | x86          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x86.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x86.rpm)
+Linux    | Debian  | x64          | 1.1.0 | [omi-`*.*.*-*`.ssl_110.ulinux.x64.deb](https://github.com/Microsoft/omi/releases/latest)
```
Comment (jeffaco) [inline README.md]: I thought I did this "right" when I first created this, but clearly my intent was not communicated.

See lines 30-32 before this. In particular:

You can download and install OMI from the [Releases] page. **While
version numbers change from release to release, the following
downloads illustrates the package types for OMI:**


It was never the intent that this file be modified from release to release. This was supposed to provide a "snapshot in time" idea of what to find. Probably all the links should just be references to the latest releases page without specific filenames.

File: README.md [hunk 2]
```diff
@@ -76,12 +76,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
 
 - For RPM based systems (RedHat, Oracle, CentOS, SuSE):
 ```
-sudo rpm -Uvh ./omi-1.4.1-0.ssl_100.ulinux.x64.rpm
+sudo rpm -Uvh ./omi-*.*.*-*.ssl_100.ulinux.x64.rpm
```
Comment (jeffaco) [inline README.md]: Again, these were supposed to be "general ideas", not "cut and paste" examples. How about doing something like:

sudo rpm -Uvh ./omi-MAJOR.MINOR.PATCH-BUILD.ssl_100.ulinux.x64.rpm

where MAJOR, MINOR, PATCH, and BUILD refer to the version number information for the specific version of OMI that you wish to install.

Something along those lines, anyway.

File: README.md [hunk 3]
```diff
@@ -76,12 +76,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
...
 ```
 
 - For DPKG based systems (Debian, Ubuntu, etc):
 ```
-sudo dpkg -i ./omi-1.4.1-0.ssl_100.ulinux.x64.deb
+sudo dpkg -i ./omi-*.*.*-*.ssl_100.ulinux.x64.deb
```
Comment (jeffaco) [inline README.md]: Ditto to comment above.

**Comment by JumpingYang001** [comment]
ok, I will update them today, thanks.

**Comment by JumpingYang001** [comment]
@jeffaco, I have fixed based on your comments, could you help to check it again for if there is anything need to update? thank you!

File: README.md [hunk 4]
```diff
@@ -76,12 +78,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
 
 - For RPM based systems (RedHat, Oracle, CentOS, SuSE):
 ```
-sudo rpm -Uvh ./omi-1.4.1-0.ssl_100.ulinux.x64.rpm
+sudo rpm -Uvh ./omi-MAJOR.MINOR.PATCH-BUILD.ssl_100.ulinux.x64.rpm
```
Comment (jeffaco) [inline README.md]: That's fine, but do you want to give an example for some specific version of OMI (say v1.3.0-2)? This is not to keep up to date, just an example. Like:

For example, if you wish to install the RPM version of OMI for v1.3.0-2, you would use a command like:

```
sudo rpm -Uvh ./omi-1.3.0-2.ssl_100.ulinux.x64.rpm
```

?

Would that be more clear, or do you think what you have is pretty clear?

To be precise, the intent is NOT to keep this version number up to date. It's just an example for a point-in-time release.

File: README.md [hunk 5]
```diff
@@ -76,12 +78,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
...
 ```
 
 - For DPKG based systems (Debian, Ubuntu, etc):
 ```
-sudo dpkg -i ./omi-1.4.1-0.ssl_100.ulinux.x64.deb
+sudo dpkg -i ./omi-MAJOR.MINOR.PATCH-BUILD.ssl_100.ulinux.x64.deb
```
Comment (jeffaco) [inline README.md]: Ditto

**Comment by JumpingYang001** [comment]
@jeffaco , I have updated it, it should be better now, thanks.

File: README.md [hunk 6]
```diff
@@ -76,14 +78,15 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
...
+sudo rpm -Uvh ./omi-1.3.0-2.ssl_100.ulinux.x64.rpm
 ```
 
 - For DPKG based systems (Debian, Ubuntu, etc):
 ```
-sudo dpkg -i ./omi-1.4.1-0.ssl_100.ulinux.x64.deb
+sudo dpkg -i ./omi-1.3.0-2.ssl_100.ulinux.x64.deb
 ```
 
+**Note:** 1.3.0-2 is just an example, you should use the specific latest release version of OMI.
```
Comment (jeffaco) [inline README.md]: s/specific latest/latest/

Comment (JumpingYang001) [reply README.md]: ok

**Pull request merged**